addons/pos_self_order/static/src/app/components/product_info_popup/product_info_popup.xml

```diff
@@ -10,8 +10,17 @@
                             <button type="button" class="btn-close" t-on-click.stop="() => this.props.close()"></button>
                         </div>
                         <span class="modal-body o_self_order_main_desc fs-3 p-4 ps-5 overflow-auto" t-out="props.product.description_self_order" />
-                        <div class="modal-footer">
+                        <div class="modal-footer d-flex flex-row-reverse justify-content-between align-items-center">
                             <button type="button" class="btn btn-primary" t-on-click.stop="() => this.addToCartAndClose()">Add to Cart</button>
+                            <div t-if="!props.product.isCombo and !props.product.attributes.length > 0" class="o_self_order_incr_button btn-group " role="group" aria-label="Quantity select" >
+                                <button type="button"
+                                    t-on-click = "() => this.changeQuantity(false)"
+                                    class="btn btn-secondary"><span class="fs-2 lh-1 fa-fw d-inline-block">－</span></button>
+                                <div class="o-so-tabular-nums d-flex align-items-center px-3 text-bg-200" t-esc="state.qty"/>
+                                <button type="button"
+                                    t-on-click = "() => this.changeQuantity(true)"
+                                    class="btn btn-secondary"><span class="fs-2 lh-1 fa-fw d-inline-block">＋</span></button>
```

Comment:
could you use [this](https://github.com/odoo/odoo/blob/master/addons/point_of_sale/static/src/app/generic_components/inputs/numeric_input/numeric_input.js#L5) component here?

Reply:
This is a good idea, I'll try to do it after the freeze